Android/app/src/main/java/app/intra/ui/MainActivity.java

```diff
@@ -300,12 +301,15 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
       tryAllButton.setEnabled(false);
       tryAllButton.setText(R.string.checking_servers);
       String[] urls = getResources().getStringArray(R.array.urls);
+      FirebaseAnalytics analytics = FirebaseAnalytics.getInstance(this);
+      analytics.logEvent(Names.TRY_ALL.name(), null);
```

Comment:
Maybe something like `TRY_ALL_REQUESTED` or `TRY_ALL_STARTED`.
It will be more explicit about what step of the TRY_ALL process it's about. One could think it's the completion of the process.

Reply:
Done